src/lib/multicall/useMulticall.ts

```diff
@@ -163,9 +169,21 @@ export function useMulticall<TConfig extends MulticallRequestConfig<any>, TResul
     },
   });
 
+  const handleMutate: KeyedMutator<TResult | undefined> = useCallback(
+    (...args) => {
+      const opts = args[1];
+      const shouldRevalidate = !(typeof opts !== "boolean" && opts && opts.revalidate === false);
```

Comment:
hard to read this condition
can we simplify it?

Reply:
spread calculation to multiple ifs and added self describing var
```ts
  let shouldRevalidate: boolean | undefined;

      if (opts === false) {
        shouldRevalidate = false;
      } else if (typeof opts !== "boolean" && opts) {
        const isRevalidateExplicitlyDisabled = opts.revalidate === false;
        shouldRevalidate = !isRevalidateExplicitlyDisabled;
      }

```